plugins/ruby-foreman-fog-proxmox/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-foreman-fog-proxmox (0.6.0-1) stable; urgency=low
+
+  * 0.6.0 released. Update fog-proxmox dependency to 0.6.0
+
+ --  <>  Tue, 16 Apr 2019 15:12:09 +0200
```

Comment:
This needs an update

Reply:
@tristanrobert Could you add your details here? I think the rest is good to go :rocket:.